HW0/P6/P6.txt

```diff
@@ -1 +1,17 @@
 # Your answers here
+P6A:
+	The output of print Jobs has different order each time the script run.
+	After several times of running, I found the order of each sentence appearing is random.
+	I think the randomness happened because there is only one input flow for the I/O. 
+	Since the burnTime function is run on 4 processor simultaneous, the processors turn the data into I/O at the same time.
+	Then the I/O randomly pick one statement to print out.
```

Comment:
I don't fully agree with this but I might be wrong. I think that a processor might take longer on the same job than another processor, for many reasons like background processes happening, the processor being busy, etc. But I don't think that the processors finish at the same time and then the I/O selects randomly